packages/now-next/src/index.ts

```diff
@@ -70,7 +70,7 @@ import {
   syncEnvVars,
   validateEntrypoint,
 } from './utils';
-import findUp from 'find-up';
+// import findUp from 'find-up';
```

Comment:
Let's remove this?

Reply:
We want to re-enable this code ASAP so figured it was just easier to leave commented out.